solidity/contracts/bridge/Bridge.sol

```diff
@@ -121,6 +130,18 @@ contract Bridge is Ownable {
     /// @notice Handle to the Bitcoin relay.
     IRelay public immutable relay;
 
+    // TODO: Documentation and initialization.
+    uint64 public redemptionDustThreshold;
+
+    // TODO: Documentation and initialization.
+    uint64 public redemptionTreasuryFee;
+
+    // TODO: Documentation and initialization.
+    uint64 public redemptionTxMaximumFeeShare;
```

Comment:
Or maybe just `redemptionTxMaximumFee`? It does not have to be a *share* if there is a single transaction redeemed.

Reply:
Done as part of https://github.com/keep-network/tbtc-v2/pull/128/commits/2da1001354b25771fc94494febb511a5bccaec5d